libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/LoginHost/SFSDKLoginHostStorage.m

```diff
@@ -38,15 +38,14 @@ @interface SFSDKLoginHostStorage ()
 
 @end
 
-// Key under which the list of login hosts will be stored in the user defaults.
 static NSString * const SFSDKLoginHostList = @"SalesforceLoginHostListPrefs";
-
-// Key for the host.
 static NSString * const SFSDKLoginHostKey = @"SalesforceLoginHostKey";
-
-// Key for the name.
 static NSString * const SFSDKLoginHostNameKey = @"SalesforceLoginHostNameKey";
 
+static NSString * const SFSDKLoginHostListLegacy = @"ChatterLoginHostListPrefs";
```

Comment:
I'd rather not have specific internal key references here such as `Chatter...`, especially since this never made it to public SDK. Let's consider a solution implemented in the app layer instead.